components/RadarChart.tsx

```diff
@@ -94,30 +103,48 @@ const RadarChart: React.FC<RadarChartProps> = ({ data1, data2 }) => {
     ],
   };
 
+  const maxValue = Math.max(
+    data1.totalCommits,
+    data1.totalIssuesCreated,
+    data1.totalPRsMerged,
+    data1.forks,
+    data2.totalCommits,
+    data2.totalIssuesCreated,
+    data2.totalPRsMerged,
+    data2.forks
+  );
+
+  const calculateStepSize = (max: number) => {
+    const idealSteps = 5; // You can adjust this number for more or fewer steps
+    let stepSize = Math.pow(10, Math.floor(Math.log10(max)));
+    
+    while (max / stepSize > idealSteps) {
+      stepSize *= 2;
+    }
+    
+    while (max / stepSize < idealSteps / 2) {
+      stepSize /= 2;
+    }
+    
+    return stepSize;
+  };
+
+  const dynamicStepSize = calculateStepSize(maxValue);
+
   const options: ChartOptions<"radar"> = {
     maintainAspectRatio: false,
     scales: {
       r: {
         angleLines: {
-          color: isDarkMode ? "rgba(255, 255, 255, 0.1)" : "rgba(0, 0, 0, 0.1)",
+          color: isDarkMode ? "rgb(0, 128, 128)" : "rgb(128, 128, 128)",
         },
         grid: {
-          color: isDarkMode ? "rgba(255, 255, 255, 0.1)" : "rgba(0, 0, 0, 0.1)",
+          color: isDarkMode ? "rgb(0, 128, 128)" : "rgb(128, 128, 128)",
         },
         suggestedMin: 0,
-        suggestedMax:
-          Math.max(
-            data1.totalCommits,
-            data1.totalIssues,
-            data1.totalPRsMerged,
-            data1.forks,
-            data2.totalCommits,
-            data2.totalIssues,
-            data2.totalPRsMerged,
-            data2.forks
-          ) + 10,
+        suggestedMax: maxValue + dynamicStepSize,
         ticks: {
-          stepSize: 10,
+          stepSize: dynamicStepSize,
```

Comment:
How about you replace this with the following code,
stepSize: Math.ceil( Math.max(data1.totalCommits, data2.totalCommits) / 10 )